polygeist/include/mlir/Dialect/Polygeist/IR/PolygeistTypes.td

```diff
@@ -25,4 +25,10 @@ class Polygeist_Type<string name, string typeMnemonic, list<Trait> traits = []>
   let mnemonic = typeMnemonic;
 }
 
+def Polygeist_StructType :
```

Comment:
Should we also add an optional `StringAttr` to host the name of the type? (even if we don't use it in codegen for now)

Reply:
We're also missing `summary` and `description` fields here

Reply:
I don't see any type able to use a optional field, do you know how to do that?
We are currently modifying from `LLVMStructType:getLiteral`, so there is no name.